packages/pyolite-kernel/py/pyolite/pyolite/__init__.py

```diff
@@ -5,47 +5,35 @@
 import sys
 import types
 
-from .patches import ensure_matplotlib_patch
-
 # Set the recursion limit, needed for altair
 # for more details, see: https://github.com/jtpio/jupyterlite/pull/113#issuecomment-851072065
 sys.setrecursionlimit(max(170, sys.getrecursionlimit()))
 
-# apply patches for available modules
-ensure_matplotlib_patch()
-
-import pyolite
-
-from .kernel import Pyolite
+termios_mock = types.ModuleType("termios")
+termios_mock.TCSAFLUSH = 2
 
-kernel_instance = Pyolite()
+fcntl_mock = types.ModuleType("fcntl")
+resource_mock = types.ModuleType("resource")
 
+sys.modules["termios"] = termios_mock
+sys.modules["fcntl"] = fcntl_mock
+sys.modules["resource"] = resource_mock
 
-# TODO: Make an ipython module mock
-class IPMock:
-    def __init__(self, kernel):
-        self.kernel = kernel
-
-
-class InteractiveShellMock:
-    pass
-
+from .patches import ensure_matplotlib_patch
 
-ip_mock = IPMock(kernel_instance)
+# apply patches for available modules
+ensure_matplotlib_patch()
 
-ip = types.ModuleType("IPython")
-ip.get_ipython = lambda: ip_mock
-ip.InteractiveShell = InteractiveShellMock
+from .display import XStream
```

Comment:
Small cosmetic thing: the `X` prefix probably comes from the `xeus-python` inspiration?

It would be fine to have it here too. Otherwise an alternative could be `Lite`?

Reply:
Yes, We will change this. The first step is to make all things work though.